CAIPs/caip-261.md

```diff
@@ -0,0 +1,384 @@
+- `trustScoreType`: Algorithm used to process the trust score;
+
+This structure can be enriched according to the trust score kind.
+
+**Peer Trust Score**
+```json
+"@context": ["https://www.w3.org/2018/credentials/v1"],
+"type": ["VerifiableCredential", "PeerTrustScorCredential"],
+"issuanceDate": "2023-11-24T12:24:42Z",
+"issuer": "did:pkh:eip155:1:0x23d86aa31d4198a78baa98e49bb2da52cd15c6f0",
+"credentialSubject":
+{
+  "id": "did:pkh:eth:0x44dc4E3309B80eF7aBf41C7D0a68F0337a88F044",
+  "trustScore": {
+    "trustScoreScope": ["SoftwareDevelopment"],
+    "value": 0.19191918793049725,
+    "result": 0.7,
+    "creationAt": "2024-02-27T10:28:00.000Z"
+  },
+  "trustScoreType": "EigenTrust"
+},
+"proof": {}
+```
+*The above example represents an assertion issued by the issuer "did:pkh:eip155:1:0x23d86aa31d4198a78baa98e49bb2da52cd15c6f0". 
+It claims a certain level of trust to the subject "did:pkh:eip155:1:0x44dc4E3309B80eF7aBf41C7D0a68F0337a88F044" in the area of software development calculated with the EigenTrust algorithm.*
+
+## Rationale
+
+### Using cryptographic keys for peers identity
+These identifiers and keys are designed to be self-verifying, which means they can prove their authenticity independently without relying on a centralized registry or authority. 
+This self-sufficiency not only enhances security by reducing potential points of failure and attack but also promotes privacy and user control. 
+Individuals and entities can manage their identities and trust relationships directly, without intermediation, facilitating a more decentralized and resilient digital ecosystem.
+
+[Decentralized identifiers][DID] using the `pkh` and `key` methods allow for the identification of account owners or trust computers in a chain-agnostic manner without the complexity of on-chain resolution.
+
+### Using content identifier for assertions
+[Content Identifiers][CID] enable anyone to deterministically generate identifiers based on the canonicalized content of a given JSON document, and store it in a compact, tamper-evident way conducive to merging, syncing, or even CRDT patterns.
+
+### Agnostic data model
+- Flexible data ranges leveraging floats type facilitating the creation of tailored user experiences,
+- Data structures have been designed to be use-case agnostic, enabling the reusability of the data across different use-cases.
+
+### Trust Modeling
+
+1. Trust in an individual or entity is based on their qualities, or their abilities; it is not binary and evolves over time,
+2. Distrust assertions allow for the capture of suspicious behaviors,
+
+### Verification
+The verification process is crucial to ensure the use of valid and up-to-date assertions, thus maintaining the trust system's reliability and accuracy. 
+This step ensures the integrity and reliability of the assertions, which is essential for supporting trust and interoperability within the ecosystem.
+
+## Test Cases

```

Comment:
Mmm, these aren't test cases so much as possible use-cases.  It might be good to mark this section as non-normative explicitly, or leave it out of the CAIP itself and just add these in, say, the eth-magicians thread promoting the CAIP?

Reply:
we use "test cases" more in the sense of "test vectors" -- I don't think you really need this section anyways since the examples provided throughout are pretty illustrative and seem (unless mircea and raphael missed any syntax nits) to be valid both as VCs and as examples of your data model?